src/js/components/stock-movement-wizard/CreateStockMovement.jsx

```diff
@@ -80,7 +80,7 @@ const FIELDS = {
     label: 'Date requested',
     attributes: {
       required: true,
-      dateFormat: 'DD/MMM/YYYY',
+      dateFormat: 'MM/DD/YYYY',
```

Comment:
Sorry about that. We can discuss this on the call tomorrow. I do like `DD/MMM/YYYY` as the date format. I just wasn't sure what you guys were expecting so I went with the more conventionally US format of `MM/DD/YYYY`.